### ISSUE TITLE
[ALM Accelerator - BUG]: CustomAzureDevOps throws error 500 when trying to access Organization

### ISSUE THREAD

**Issue by Masciuniria** [microsoft/coe-starter-kit#7001]
### Describe the issue

When testing the GetOrganizations operation in CustomAzureDevOps component I got this issue:
{
  "error": {
    "code": 500,
    "message": "Variable organization has no value.",
    "source": "europe-002.azure-apim.net",
    "path": "choose[3]\\when[1]",
    "policyId": "",
    "clientRequestId": "a4ec6102-bd54-4f4a-b326-2616f413c1c2"
  }
}

_@mikefactorial in https://github.com/microsoft/coe-starter-kit/issues/5406#issuecomment-1530000466 posted that the issue was solved but I'm using the last ALM Accelerator version (1.0.20231012.1) and I got the same error_
__________________

I even tried using different api-versions and with memberId value and got always the same error

### Expected Behavior

Status should be 200

### What component are you experiencing the issue with?

ALM Accelerator Custom Conenctor

### What solution version are you using?

1.0.20231012.1

### Steps To Reproduce

Open the CustomAzureDevOps component
Set Security tab as explained in the documentation [here](https://learn.microsoft.com/en-us/power-platform/guidance/alm-accelerator/setup-admin-tasks#configure-the-devops-custom-connector)
Update the connector
Test the getOrganization operation
__________________
Security settings:
Authentication type: OAuth 2.0
Identity provider: Azure Active Directory
Client ID:	ApplicationId of the AppRegistration I created
Client secret: Client secret value created for the AppRegistration
Tenant ID: TenantId of my organization
Resource URL: ClientId of Azure DevOps AppRegistration copied from "APIs my organization uses" tab on Azure Portal.
Redirect URL: https://global.consent.azure-apim.net/redirect

AB#616

AB#1604

**Comment by FabianAckeret** [comment]
Hi @Masciuniria 

In your M365 Admin Center --> Settings --> Org settings --> Organization information, do you have everything filled out?

**Comment by mikefactorial** [comment]
@Masciuniria are any of the other custom connector operations working? 

One thing to check.
Make sure there is a Policy on the custom connector called "Set host to https://app.vssps.visualstudio.com" that is associated with the GetOrganizations and GetCurrentUserProfile operations 


![Image](https://github.com/microsoft/coe-starter-kit/assets/42348035/bd72274a-d213-4f8c-b08a-3b3701a3d5dc)

**Comment by Masciuniria** [comment]
> Hi @Masciuniria
> 
> In your M365 Admin Center --> Settings --> Org settings --> Organization information, do you have everything filled out?

@FabianAckeret Just tried but didn't work :(

**Comment by Masciuniria** [comment]
> @Masciuniria are any of the other custom connector operations working?
> 
> One thing to check. Make sure there is a Policy on the custom connector called "Set host to https://app.vssps.visualstudio.com" that is associated with the GetOrganizations and GetCurrentUserProfile operations
> 
> ![Image](https://user-images.githubusercontent.com/42348035/281868227-bd72274a-d213-4f8c-b08a-3b3701a3d5dc.png)

@mikefactorial Just checked now. None of the six policies were associated with any operation. Do you think this is an incorrect setting of the custom connector?

In any case, I have now set the policy called "Set host to https://app.vssps.visualstudio.com" on the two operations you indicated but this action had no impact on the response. I keep getting the same error.
{
  "error": {
    "code": 500,
    "message": "Variable project has no value.",
    "source": "europe-002.azure-apim.net",
    "path": "choose[4]\\when[1]",
    "policyId": "",
    "clientRequestId": "3e8fcfe4-a1dd-4367-8fa4-c00d066cee0a"
  }
}

Other suggestions?

**Comment by Masciuniria** [comment]
@mikefactorial could you please tell me how the policy "Set host to https://vssps.dev.azure.com" should be set? to which operations should be applied?

**Comment by R4isin** [comment]
@Masciuniria, I don't know if you already check the documentation but you can find the details about the Policy here: https://learn.microsoft.com/en-us/power-platform/guidance/alm-accelerator/setup-upgrade-configuration#verify-the-azure-devops-custom-connector-policies

Remark for the Product Team, it seems that there is a gap between the documentation and the required setting for the second Policy "Set host to https://app.vssps.visualstudio.com". In the documentation, the operation is "GetUserCurrentProfil" is missing. And without this operation, an error notification is showed when we launch the app.

Kr,

**Comment by RajeevPentyala** [comment]
@Masciuniria Please refer the [doc](https://learn.microsoft.com/en-us/power-platform/guidance/alm-accelerator/setup-upgrade-configuration#verify-the-azure-devops-custom-connector-policies) and let me know if you still require assistance in setting up policies.
Thanks @R4isin for your comment and feedback. We are currently working on updating the documentation and will take your feedback into consideration.

**Comment by Masciuniria** [comment]
Thank you a lot @R4isin for your comment and thank you @RajeevPentyala for offering your assistance. I didn't know about  that policies guidance in the documentation because it is part of the _Upgrade the ALM Accelerator_ chapter which I haven't reached yet.
Now I did all the changes to the policies settings and I finally got some 200 responses. Just tested the following:
- [x] GetOrganizations (Set host to https://app.vssps.visualstudio.com policy)
- [x] GetIdentities, GetUsers (https://vssps.dev.azure.com/ policy)
- [x] GetProjects, GetProject, GetProjectTeams, GetProjectTeamMembers (no policy)
